src/main/java/net/dries007/tfc/util/DispenserBehaviors.java

```diff
@@ -152,6 +182,50 @@ protected void playSound(BlockSource source)
         }
     };
 
+    public static final DispenseItemBehavior TFC_HANDSTONE_BEHAVIOR = new DefaultDispenseItemBehavior()
+    {
+        @Override
+        public ItemStack execute(BlockSource source, ItemStack stack)
+        {
+            final Level level = source.getLevel();
+            final BlockPos pos = source.getPos().relative(source.getBlockState().getValue(DispenserBlock.FACING));
+            if (Helpers.isItem(stack, TFCItems.HANDSTONE.get()) && level.getBlockState(pos).getBlock() instanceof QuernBlock)
+            {
+                return level.getBlockEntity(pos, TFCBlockEntities.QUERN.get()).filter(quern -> !quern.hasHandstone()).flatMap(quern ->
+                    quern.getCapability(Capabilities.ITEM).map(item -> {
+                            item.insertItem(QuernBlockEntity.SLOT_HANDSTONE, stack, false);
+                            return ItemStack.EMPTY;
+                        }
+
+                    )
```

Comment:
```suggestion
                    quern.getCapability(Capabilities.ITEM).map(item -> {
                        item.insertItem(QuernBlockEntity.SLOT_HANDSTONE, stack, false);
                        return ItemStack.EMPTY;
                    })
```
formatting is a bit off